daemon/config/config_unix.go

```diff
@@ -8,13 +8,20 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/opts"
 	units "github.com/docker/go-units"
+	"github.com/docker/libnetwork/ipamutils"
 )
 
 const (
 	// DefaultIpcMode is default for container's IpcMode, if not set otherwise
 	DefaultIpcMode = "shareable" // TODO: change to private
 )
 
+// NetworkConfig stores the daemon-wide networking configurations
+type NetworkConfig struct {
+	// Default address pools for docker networks
+	DefaultAddressPools []*ipamutils.NetworkToSplit `json:"default-node-local-address-pools,omitempty"`
```

Comment:
It looks like this is identical in both unix and windows. Can this go into `CommonConfig` instead?

Reply:
Currently we dont support it on windows.

Reply:
I see you adding this same field in `config_windows.go`. Should it be removed then?

Reply:
My CI for windows p *windowsRS1  * failed due to build issue. so I had to
modify to fix CI failure.

Reply:
Either way there is a problem here.

Either the code needs to be changed so it's not referenced on windows, or this field needs to be moved into a non-platform specific file. It does not make sense to dupliacte it.